FaunaDB.Client/Client/DefaultClientIO.cs

```diff
@@ -27,25 +27,27 @@ class DefaultClientIO : IClientIO
         readonly AuthenticationHeaderValue authHeader;
 
         private LastSeen lastSeen;
+        private Version httpVersion;
 
         public const string StreamingPath = "stream";
         public const HttpMethodKind StreamingHttpMethod = HttpMethodKind.Post;
 
-        internal DefaultClientIO(HttpClient client, AuthenticationHeaderValue authHeader, LastSeen lastSeen, Uri endpoint, TimeSpan? timeout)
+        internal DefaultClientIO(HttpClient client, AuthenticationHeaderValue authHeader, LastSeen lastSeen, Uri endpoint, TimeSpan? timeout, Version httpVersion)
         {
             this.client = client;
             this.authHeader = authHeader;
             this.lastSeen = lastSeen;
             this.endpoint = endpoint;
             this.clientTimeout = timeout;
+            this.httpVersion = httpVersion == null ? new Version(1, 1) : httpVersion;
```

Comment:
can't we default to http2?

Reply:
thanks for the suggestion, I've changed it to Version(2, 0) as a default for .net standard 2.1